algorithmic_efficiency/logger_utils.py

```diff
@@ -272,15 +272,11 @@ def get_meta_data(workload: spec.Workload) -> dict:
   meta_data.update(system_software_info)
   system_hardware_info = _get_system_hardware_info()
   meta_data.update(system_hardware_info)
+  if rng_seed:
```

Comment:
I think this should be `if rng_seed is not None:` because `rng_seed` could be `0` and then this condition what also evaluate to `False`.

Reply:
That is a good point, will update this thanks!